packages/okta-adapter/e2e_test/adapter.test.ts

```diff
@@ -437,19 +438,28 @@ const getChangesForInitialCleanup = async (
   types: ObjectType[],
   client: OktaClient,
 ): Promise<Change<InstanceElement>[]> => {
-  const cleanupChanges: Change<InstanceElement>[] = []
-  cleanupChanges.concat(
+  let cleanupChanges: Change<InstanceElement>[] = []
```

Comment:
There's a preferences to avoid `let`
so maybe something like - 
```suggestion
const removalChanges = element.filter(....)
const brandChanges = (await createBrandChangesForDeploy(types, client)).filter(...)
const cleanupChanges = removalChanges.concat(brandChanges)
```